src/SmartComponents/Notifications/notifications.scss

```diff
@@ -2,6 +2,7 @@
   position: fixed;
   top: 0;
   right: 0;
+  z-index: 100;
```

Comment:
Too low number, main content has z-index set to `100` so notifications would be still hidden.